packages/nuxt-locokit/playground/pages/keepMe.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <h1>Keep me</h1>
```

Comment:
Why ?

Reply:
Because nuxt (beta version) doesn't display our custom Error page without a page in the project. Maybe it is fix or it will be in the next version.

Reply:
Ok, do you want to check it in this PR or in an other issue ?
If you prefer in an other issue, please create one and reference it here.